src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -427,12 +464,18 @@ slots:
     range: EntityReference
     examples:
       - value: MONDO_MAPPINGS:mondo_exactmatch_ncit.sssom.tsv
+        description: A reference to the mapping set that originally contained this mapping.
   mapping_cardinality:
     description: A string indicating whether this mapping is from a 1:1 (the subject_id
       maps to a single object_id), 1:n (the subject maps to more than one object_id), 
       n:1, 1:0, 0:1 or n:n group. Note that this is a convenience field that should be derivable 
       from the mapping set.
     range: mapping_cardinality_enum
+    examples:
+      - value: "1:1"
```

Comment:
can you make this exhaustive?

and more specifically, give examples of one-to-one and one-to-many mappings, with an explanation of why they have these qualities

Reply:
The enum definition is exhaustive; the examples are just cherry-picked. You can see all the enums and what they mean on the enum page (which is linked from the slot/field/column page).